src/Elements/HrefVariablesElement.php

```diff
@@ -15,7 +15,7 @@ public function getAllVariables()
             $memberContent = $member->getContent();
 
             $hrefVariable = new HrefVariable();
-            $hrefVariable->description = $member->getMetaData()['description'];
+            $hrefVariable->description = empty($member->getMetaData()['description']) ? '' : $member->getMetaData()['description'];
```

Comment:
Please use PHP7's ` ?? ''` instead of `empty` and the ternary operator. Like, for example, in PR #9

Reply:
good call, updated